src/core/storage/preferences/preferencesStorage.ts

```diff
@@ -9,6 +9,8 @@ enum PreferencesKeys {
   APP_ALREADY_INIT = "app-already-init",
   APP_STATE_FLAGS = "app-state-flags",
   APP_LANGUAGE = "app-language",
+  APP_DEFAULT_CRYPTO_ACCOUNT = "app-default-crypto-account",
+  APP_HIDE_CRYPTO_BALANCES = "app-hide-crypto-balances",
```

Comment:
`APP_DEFAULT_CRYPTO_ACCOUNT` and `APP_HIDE_CRYPTO_BALANCES` can be removed - this got back in with the merge conflict changes :)